api_tests/users/views/test_user_files_list.py

```diff
@@ -130,3 +130,11 @@ def test_get_files_has_links(self, app, user, url):
 
         assert 'upload' in file_detail_json['links']
         assert file_detail_json['links']['upload'] == waterbutler_url
+
+    def test_disabled_users_quickfiles_gets_410(self, app, url):
```

Comment:
Minor: you can use the fixtures provided here (`quickfiles`, `url`, `user`) if you set `user.is_disabled = True` at the beginning of this test.